grafana-plugin/src/components/ScheduleBorderedAvatar/ScheduleBorderedAvatar.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import cn from 'classnames/bind';
+
+import styles from './ScheduleBorderedAvatar.module.scss';
+
+const cx = cn.bind(styles);
+
+export default function ScheduleBorderedAvatar({
+  colors,
+  renderAvatar,
+  renderIcon,
+  width,
+  height,
+}: {
+  colors: string[];
+  width: number;
+  height: number;
+  renderAvatar: () => JSX.Element;
+  renderIcon: () => JSX.Element;
+}) {
+  return <div className={cx('root')}>{colors?.length ? renderSVG() : renderAvatarIcon()}</div>;
+
+  function renderAvatarIcon() {
+    return (
+      <>
+        <div className={cx('avatar')}>{renderAvatar()}</div>
+        <div className={cx('icon')}>{renderIcon()}</div>
+      </>
+    );
+  }
+
+  function renderSVG() {
+    return (
+      <>
+        <svg xmlns="http://www.w3.org/2000/svg" version="1.1" width={width} height={height} viewBox="-10 -10 220 220">
+          <g fill="none" strokeWidth="35" transform="translate(100,100)">
```

Comment:
hmm, why exactly these numbers `-10 -10 220 220`? Can you please declare some number constants and use them. For example :
```
const d = `M {lastX.toFixed(3)},{lastY.toFixed(3)} A {radius},{radius} 0 0,1 {x.toFixed(3)},{y.toFixed(3)}`;
```

I'm not sure is it correct in this case, I mean general idea - use understandable constants in math expressions